src/app/pages/Story/__snapshots__/index.test.jsx.snap

```diff
@@ -21,12 +21,6 @@ exports[`Story Page should not show the pop-out timestamp when allowDateStamp is
   font-weight: inherit;
```

Comment:
If we're introducing a new prop I would expect to see at least one more test?

Reply:
Yeah makes sense. But would we put it in the [`CpsRelatedContent`](https://github.com/bbc/simorgh/blob/latest/src/app/containers/CpsRelatedContent/index.test.jsx) test as that's where it's introduced?